### PR TITLE
Read enableLeakCanary from developer gradle properties

### PR DESCRIPTION
**Pull request by kidinov**
### Description

Allows to declare `ext.enableLeakCanary` in `init.gradle` file

### Testing information

* Modify `~/.gradle/init.gradle` with `ext.enableLeakCanary=false`
* Add `println "Enable LeakCanary: ${isLeakCanaryEnabled()}"` to `build.gradle`
* Notice correct value printed

### The tests that have been performed

Above

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

### CODE REVIEW

File: WooCommerce/build.gradle [hunk 1]
```diff
@@ -567,9 +567,7 @@ static def loadPropertiesFromFile(inputFile) {
 }
 
 def isLeakCanaryEnabled() {
-    return project.properties.getOrDefault('enableLeakCanary', 'true') == 'true'
+    return gradle.hasProperty('enableLeakCanary') && gradle.enableLeakCanary == true
```
Comment (wzieba) [inline WooCommerce/build.gradle]: Thanks for this change @kidinov , I forgot about this property!

WDYT about using `developers.properties` for this? It looks like a developer config indeed.

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/woocommerce/woocommerce-android/milestone/244">21.2</a>. This milestone is due in less than 2 days.<br />
Please make sure to get it merged by then or assign it to a milestone with a later deadline.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WooCommerce/build.gradle (hunk 1 above)
Comment (wzieba) [reply WooCommerce/build.gradle]: What I mean is something like this: 795879d0cf1e7783e49581a09dce185cccbdaf42

Comment (kidinov) [reply WooCommerce/build.gradle]: That's even better! I updated the branch + I made it true by default in the example file

Comment (wzieba) [reply WooCommerce/build.gradle]: Thanks! Right, it's `true` by default on `trunk` 👍

**Review by wzieba — APPROVED** [review]

**Pull request merged**